wiki/Community/osu!monthly/en.md

```diff
@@ -0,0 +1,53 @@
+# osu!monthly
+
+**osu!monthly** is a monthly newsletter managed by the [osu!monthly Team](/wiki/People/osu!monthly_Team), with the goal of summarising the various community and official events, news, announcements, etc. that occur every month. The project has been led by several other groups of community members in the past, with the goal always being the same: inform the average player about the previous and current events in order to shed light over the entire osu! community.
+
+## Becoming an osu!monthly contributor
+
+osu!monthly is a community effort with many people contributing to its creation. Anyone that wants to help or contribute, in ways such as writing, sharing opinions and thoughts, or providing feedback, is always welcome to do so. The [osu!monthly Team](/wiki/People/osu!monthly_Team) consists of such contributors who consistently help out with the project.
+
+All discussion and project organisation takes place in the `#osu-news` channel of the [osu!dev Discord server](/wiki/Community/osu!dev_Discord_server).
+
+## History
+
+The original run of osu!monthly created by [jjrocks](https://osu.ppy.sh/users/30985), [Corin](https://osu.ppy.sh/users/639270), and friends ran from March 2011 to October 2011. The magazine was then released in a `.pdf` format with all the most important information about the osu! community.
+
+Inspired by the first version of the osu!monthly, a new team was formed thanks to [mangomizer](https://osu.ppy.sh/users/1893718) and [clayton](https://osu.ppy.sh/users/3666350) and new iterations of the monthly could be published from October 2017 to February 2018. The format was still `.pdf` at the time, but with a more consistent and pleasing design.
+
+With a desire to once again revive this project, the [osu!monthly Team](/wiki/People/osu!monthly_Team) was formed with [0x84f](https://osu.ppy.sh/users/7944724) at the helm, and relaunched the project this time directly as news posts, and not by `.pdf`. The project is currently ongoing.
```

Comment:
think you don't need to capitalize the team word here (also the pdf bit is redundant, again)
```suggestion
With a desire to once again revive this project, the [osu!monthly team](/wiki/People/osu!monthly_Team) was formed with [0x84f](https://osu.ppy.sh/users/7944724) at the helm, and relaunched the project, this time directly as news posts. The project is currently ongoing.
```